app/javascript/react/components/SessionFilters/TagsInput.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from "react";
+
+import { useCombobox } from "downshift";
+import { useTranslation } from "react-i18next";
+import { useAppDispatch, useAppSelector } from "../../store/hooks";
+import { setLoading } from "../../store/mapSlice";
+import { fetchTags, selectTags } from "../../store/sessionFiltersSlice";
+import { SessionTypes, fetchTagsParamsType } from "../../types/filters";
+import { UrlParamsTypes, useMapParams } from "../../utils/mapParamsHandler";
+import { FilterInfoPopup } from "./FilterInfoPopup";
+import * as S from "./SessionFilters.style";
+
+const TagsInput = () => {
+  const [items, setItems] = useState([""]);
+  const [inputValue, setInputValue] = useState("");
+  const [selectedItem, setSelectedItem] = useState("");
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation();
+  const {
+    setUrlParams,
+    tags,
+    boundWest,
+    boundEast,
+    boundNorth,
+    boundSouth,
+    usernames,
+    initialSensorName,
+    initialUnitSymbol,
+    sessionType,
+  } = useMapParams();
+
+  const tagsToSelect = useAppSelector(selectTags);
+
+  const fixedSessionTypeSelected = sessionType === SessionTypes.FIXED;
+  const selectedSessionType = sessionType || SessionTypes.FIXED;
+
+  // Filters (temporary solution)
+  const sensorName = fixedSessionTypeSelected
+    ? initialSensorName
+    : "AirBeam-PM2.5";
+
+  const { isOpen, getMenuProps, getInputProps, getItemProps, reset } =
+    useCombobox({
+      items: tagsToSelect,
+      inputValue,
+      selectedItem,
+      onInputValueChange: ({ inputValue }) => {
```

Comment:
same suggestion as in `ProfileNamesInput` - it might be a good idea to provided default values to prevent potential undefined values in `onInputValueChange`